src/MinFraud/Model/Email.php

```diff
@@ -13,7 +13,8 @@
  * Mail.
  * @property-read bool|null $isHighRisk This field is true if MaxMind believes
  * that this email is likely to be used for fraud. Note that this is also
- * factored into the overall risk_score in the response as well.
+ * factored into the overall risk_score in the response as well. Deprecated
+ * effective August 29th, 2019.
```

Comment:
Maybe bold deprecated here and below.

Reply:
Also, minor style nit, drop the `th` on the date.

Reply:
I just realized that this is the wrong `is_high_risk`. Could you make a new PR to fix it?